Configuring --without-prrte fails

With the recent additions of the prrte man page content into mpirun(1), something isn't quite right when configuring `--without-prrte`; building results in the following error:

```
make[1]: warning: -jN forced in submake: disabling jobserver mode.
  MKDIR    prrte-rst-content
  MKDIR    schizo-ompi-rst-content
if test ! -d "$dir"; then mkdir "$dir"; fi
mkdir: .: No such file or directory
make[1]: *** [schizo-ompi-rst-content/schizo-ompi-cli.rstxt] Error 1
make: *** [all-recursive] Error 1
```

--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -962,10 +962,9 @@ $(builddir)/schizo-ompi-rst-content/schizo-ompi-cli.rstxt: $(OMPI_SCHIZO_OMPI_RS
 else
 $(builddir)/schizo-ompi-rst-content/schizo-ompi-cli.rstxt: $(builddir)/schizo-ompi-rst-content
 $(builddir)/schizo-ompi-rst-content/schizo-ompi-cli.rstxt: $(srcdir)/no-prrte-content.rst.txt
-	if test ! -d "$$dir"; then mkdir "$$dir"; fi
+	dir=`dirname $@`; if test ! -d "$$dir"; then mkdir "$$dir"; fi
 	$(OMPI_V_SPHINX_COPYRST) \
-	dir=`dirname $@`; \
-	cp -pf $(srcdir)/no-prrte-content.rst.txt "$$dir"
+	cp -pf $(srcdir)/no-prrte-content.rst.txt "$@"
 endif
 
 $(ALL_MAN_BUILT): $(builddir)/prrte-rst-content

